src/components/atoms/imgs/StyledImg.tsx

```diff
@@ -0,0 +1,25 @@
+interface MyComponentProps {
+  children?: React.ReactNode;
+  src: any;
+  alt: string;
+  zoomMid?: boolean;
```

Comment:
yes, zoom middle (which is the default behavior, when using "object-fit: cover")... But with lineUp images I got, the majority require to zoom top, just in case of 2 bands it is better the keep default behavior - zoomMid.